e2e/common/createCaseHelper.ts

```diff
@@ -0,0 +1,89 @@
+import { APIRequestContext, expect, request } from "@playwright/test";
+import fs, { existsSync } from "fs";
+import path from "path";
+import { getAccessToken } from "./getAccessTokenHelper";
+import jsonData from "../caseData/citizenDA/courtNavDaCitizenCase.json";
+
+/**
+ * Function to create a DA Citizen CourtNav case and optionally add a document.
+ * @param {boolean} withDoc Whether to add a document after case creation
+ */
+async function createDaCitizenCourtNavCase(withDoc: boolean): Promise<void> {
+  const apiContextDaCreateCase: APIRequestContext = await request.newContext();
+  const tokenDaCreateCase = await getAccessToken(
+    "daCourtNavCreateCase",
+    apiContextDaCreateCase,
+  );
+  if (!tokenDaCreateCase) {
+    throw new Error("Setup failed: Unable to get bearer token.");
+  }
+  process.env.COURTNAV_CREATE_CASE_BEARER_TOKEN = tokenDaCreateCase;
+  try {
+    const response = await apiContextDaCreateCase.post(
+      process.env.COURTNAV_CASE_URL as string,
+      {
+        headers: {
+          Authorization: `Bearer ${tokenDaCreateCase}`,
+          "Ocp-Apim-Subscription-Key": process.env
+            .COURTNAV_SUBSCRIPTION_KEY_CREATE_CASE as string,
+        },
+        data: jsonData,
+      },
+    );
+    const responseBody = await response.json();
+    const ccd_reference = responseBody.ccd_reference as string;
```

Comment:
if responseBody is null, it will throw NPE, maybe modify if condition as 

 if (!responseBody && !responseBody.ccd_reference) {